src/content/1.7/faq/upgrade.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Upgrade FAQ
+---
+
+# Upgrade FAQ
+
+- [After upgrading, I lost access to some Back-Office pages](#restore-access-to-back-office-page)
+
+
+## Restore access to Back Office Page
+
+**Q:** After upgrading my PrestaShop to a new version, I lost access to some Back-Office pages. How can I fix it?
+
+**A:** It is likely that some SQL configuration is not correct.
+
+First, identify what is the `slug` of the Back Office pages. You can find them into SQL table `ps_authorization_role`. This will tell you the SQL identifier for these pages.
```

Comment:
```suggestion
First, identify what is the `slug` of the Back Office pages. You can find them into the SQL table `ps_authorization_role`. This will tell you the SQL identifier for these pages.
```